api/impl/address.go

```diff
@@ -110,21 +115,15 @@ func (api *nodeAddress) Export(ctx context.Context, addrs []address.Address) ([]
 	return out, nil
 }
 
-func parseKeyInfos(d files.Directory) ([]*types.KeyInfo, error) {
-	iter := d.Entries()
-	var kinfos []*types.KeyInfo
-	for iter.Next() {
-		fi, ok := iter.Node().(files.File)
-		if !ok {
-			return nil, fmt.Errorf("file passed to import was not a file")
-		}
-
-		var ki types.KeyInfo
-		if err := json.NewDecoder(fi).Decode(&ki); err != nil {
-			return nil, err
-		}
+// WalletImportResult is the structure used to hold imported wallet data.
```

Comment:
Do I understand correctly that this type defines the schema expected for imported wallets? If so make that very explicit, and probably raise this higher up in the file, above the funcs.

Reply:
The `WalletImportResult` type and [this type](https://github.com/filecoin-project/go-filecoin/blob/master/commands/address.go#L171) are the same -- I think there should be 1 type both commands use. The type cannot live in `commands/` as that will cause an import cycle with `api/impl`. I suppose I can move it to `api/impl` and import that into `commands/`